get-all-certs.ps1

```diff
@@ -1,3 +1,40 @@
+function Set-CaCerts {
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'Low')]
+    param(
+        [Parameter()]
+        [string]
+        $CertOutPath = "$env:USERPROFILE\all-certificates\cacerts.pem",
+        [Parameter()]
+        [switch]
+        $FailFast
+    )
+    #Requires -Version 6.0
```

Comment:
lets remove this requirement, please?
I checked it with 5.1 and the script is working. I hope it will work even with below versions )

Reply:
Done